### ISSUE TITLE
Uncaught TypeError: Cannot read property 'testSync' of undefined

### ISSUE THREAD

**Issue by albireox** [DSpeckhals/python-indent#54]
[Enter steps to reproduce:]

1. On a Python document write a statement that includes arguments between parenthesis.
2. Put the cursor just after the open parenthesis and press enter. The line should be broken and the continuation line indented. Instead, the error is raised and the continuation line remains unindented.

**Atom**: 1.23.0-beta0 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [python-indent](https://github.com/DSpeckhals/python-indent) package 1.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'testSync' of undefined

```
At /Users/albireo/.atom/packages/python-indent/lib/python-indent.js:322

TypeError: Cannot read property 'testSync' of undefined
    at PythonIndent.indentHanging (/packages/python-indent/lib/python-indent.js:322:19)
    at PythonIndent.indent (/packages/python-indent/lib/python-indent.js:37:18)
    at HTMLElement.editorNewline (/packages/python-indent/lib/main.js:23:57)
    at CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:388:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:108:40)
```

### Commands

```
  3x -1:03.0 core:move-left (input.hidden-input)
     -1:02.2.0 editor:newline (input.hidden-input)
     -1:01.5.0 core:move-down (input.hidden-input)
 18x -1:01.4.0 core:move-right (input.hidden-input)
  7x -1:00.2.0 core:move-left (input.hidden-input)
     -0:59.2.0 core:move-right (input.hidden-input)
 16x -0:50.5.0 core:backspace (input.hidden-input)
     -0:46.6.0 editor:move-to-first-character-of-line (input.hidden-input)
     -0:46 snippets:next-tab-stop (input.hidden-input)
     -0:46 snippets:expand (input.hidden-input)
     -0:46 editor:indent (input.hidden-input)
     -0:45.6.0 core:save (input.hidden-input)
  5x -0:45.1.0 core:backspace (input.hidden-input)
     -0:43.2.0 editor:newline (input.hidden-input)
  4x -0:05.4.0 core:backspace (input.hidden-input)
     -0:03.7.0 editor:newline (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.6 
atom-ide-ui 0.5.4 
atom-jinja2 0.6.0 
auto-detect-indentation 1.3.0 
autocomplete-clang 0.11.4 
autocomplete-emojis 2.5.0 
autocomplete-paths 2.12.1 
autocomplete-python 1.10.3 
autocomplete-python-jedi 1.10.3-0 
autocomplete-system-paths 1.0.0 
busy-signal 1.4.3 
file-header 1.13.7 
file-icons 2.1.13 
file-type-icons 1.3.4 
git-plus 7.10.0 
git-time-machine 1.5.9 
highlight-selected 0.13.1 
ide-python 0.2.3 
imdone-atom 2.2.7 
imdone-atom-github 0.1.11 
intentions 1.1.5 
language-cython 0.3.0 
language-fish-shell 1.0.5 
language-idl 0.4.0 
language-latex 1.1.1 
language-markdown 0.25.1 
language-restructuredtext 1.1.0 
language-viml 1.1.6 
last-cursor-position 0.9.2 
latex 0.49.0 
latex-autocomplete 1.0.0 
latex-completions 0.3.3 
linter 2.2.0 
linter-clang 4.1.2 
linter-coffeescript 1.0.0 
linter-flake8 2.2.1 
linter-pydocstyle 0.4.7 
linter-pylint 2.1.0 
linter-spell 0.15.0 
linter-spell-rst 0.4.0 
linter-ui-default 1.6.10 
markdown-preview-plus 2.4.16 
markdown-writer 2.7.3 
merge-conflicts 1.4.5 
multi-wrap-guide 1.0.0 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
nucleus-dark-ui 0.12.3 
open-terminal-here 2.3.1 
organized 0.8.1 
project-manager 3.3.5 
python-indent 1.1.2 
python-tools 0.6.9 
revert-buffer 0.6.0 
split-diff 1.5.1 
Sublime-Style-Column-Selection 1.7.4 
tabs-to-spaces 1.0.3 
toggle-quotes 1.0.1 
```

**Comment by DSpeckhals** [comment]
This looks like a deprecated API on the beta channel. I'll look into it.

**Comment by DSpeckhals** [comment]
This was fixed in #55 and the package was published. Thanks for filing the issue!
